docs/guides/backend_guide.md

```diff
@@ -0,0 +1,138 @@
+---
+page_title: "Using Backend Guide"
+description: |-
+Using your Scaleway DBaaS as to store your terraform state.
+---
+
+# Set up your database with the Scaleway provider
+
+Before initializing the backend with `terraform init`, the database must already exist:
```

Comment:
You should start with something like:

First, let's create a state locally, then we will transfer the state to a remote database.
If you already have a remote database existing you can already go to step X